### PR TITLE
Add some missing style props

### LINKED ISSUE #240: Support justify-items
Thanks for all the thoughtful effort here!

I was working today and found support for the `justify-items` property is missing. Perhaps it is intentional or I am confused somehow, but I thought it worth flagging.

### PR DESCRIPTION
**Pull request by flybayer**
This adds `fontStyle`, `overflow`, `justifyItems`, `gridTemlateAreas`, and `gridArea`.

Closes #240 

If you want me to rebuild the docs for this PR, I can do so.

### CODE REVIEW

**Comment by jxnblk** [comment]
Thanks for this! I think the tests are only failing because of an update in Jest that essentially causes all the tests to fail. (The solution is adding `"testURL": "http://localhost"` to the jest config – I can handle this later)

In order to get this in, the docs will need to be updated as well, if you wanna take a stab at that

**Comment by flybayer** [comment]
@jxnblk I added docs and fixed the tests!

File: docs/table.md [hunk 1]
```diff
@@ -68,6 +69,7 @@ Function Name | Prop       | CSS Property    | Theme Field
 `gridColumnGap` | `gridColumnGap` | `grid-column-gap` | `space`
 `gridColumn` | `gridColumn` | `grid-column` | none
 `gridRow` | `gridRow` | `grid-row` | none
+`gridArea` | `gridArea` | `grid-area` | none
```
Comment (Lavoaster) [inline docs/table.md]: I think you're missing `gridTemplateAreas` here.

File: system-components/package.json [hunk 2]
```diff
@@ -44,6 +44,7 @@
     "roots": [
       "<rootDir>"
     ],
+    "testURL": "http://localhost",
```
Comment (Lavoaster) [inline system-components/package.json]: Jest `23.5.0` came out a few days ago which resolved the need to have this in the config. But it's also harmless to have.

File: docs/table.md (hunk 1 above)
Comment (flybayer) [reply docs/table.md]: You are right, thanks for the catch!

**Comment by Lavoaster** [comment]
@jxnblk Is there anything preventing this from being pulled in?

**Pull request merged**